generate-plugin.js

```diff
@@ -0,0 +1,134 @@
+'use strict'
+
+const {
+  readFile,
+  writeFile
+} = require('fs').promises
+const { existsSync } = require('fs')
+const path = require('path')
+const chalk = require('chalk')
+const generify = require('generify')
+const argv = require('yargs-parser')
+const cliPkg = require('./package')
+const { execSync } = require('child_process')
+const log = require('./log')
+
+const pluginTemplate = {
+  dir: 'plugin',
+  main: 'index.js',
+  types: 'index.d.ts',
+  scripts: {
+    lint: 'standard && npm run lint:typescript',
+    'lint:typescript': 'standard --parser @typescript-eslint/parser --plugin @typescript-eslint/eslint-plugin test/types/*.ts',
```

Comment:
Shall we put these configurations in a dedicated `.eslintrc` file?

Reply:
After further investigation I found out that `standard` has some issues with TS even using the parser and the plugin.
I found [ts-standard](https://github.com/toddbluhm/ts-standard) which looks good and needs no config but works out of the box.
This way I could also remove those dependencies and we need no more configuration.
What do you think?